app/src/ui/changes/filter-changes-list.tsx

```diff
@@ -1123,32 +1116,34 @@ export class FilterChangesList extends React.Component<
     const disableAllCheckbox =
       files.length === 0 || isCommitting || rebaseConflictState !== null
 
+    return (
+      <div
+        className="header"
+        onContextMenu={this.onContextMenu}
+        ref={this.headerRef}
+      >
+        <Checkbox
+          ref={this.includeAllCheckBoxRef}
+          label={filesDescription}
+          value={includeAllValue}
+          onChange={this.onIncludeAllChanged}
+          disabled={disableAllCheckbox}
+          ariaDescribedBy="changesDescription"
+        />
+      </div>
+    )
+  }
+
+  public render() {
+    const { workingDirectory, isCommitting } = this.props
+    const { files } = workingDirectory
+
+    const filesPlural = files.length === 1 ? 'file' : 'files'
+    const filesDescription = `${files.length} changed ${filesPlural}`
+
     return (
       <>
         <div className="changes-list-container file-list">
-          <div
-            className="header"
-            onContextMenu={this.onContextMenu}
-            ref={this.headerRef}
-          >
-            <TooltippedContent
-              tooltip={selectedChangesDescription}
-              direction={TooltipDirection.NORTH}
-              openOnFocus={true}
-            >
```

Comment:
Tooltip is removed in favor of https://github.com/desktop/desktop/pull/19758 

This was a design decision when first POC ideas were discussed. After this entire POC is complete and merged, we can discuss if we like this direction or want to iterate. :)